### PR TITLE
[tests-only] update issue tags

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
Updated the issue tags and test coverage for etag in federation sharing

## Related Issue
- https://github.com/owncloud/ocis/issues/9813
- https://github.com/owncloud/ocis/issues/9568
- https://github.com/owncloud/ocis/issues/9534 : https://github.com/cs3org/reva/pull/4823

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by PrajwolAmatya — APPROVED** [review]

**Review by amrita-shrestha — APPROVED** [review]
lgtm

File: tests/acceptance/features/apiOcm/share.feature [hunk 1]
```diff
@@ -85,6 +87,10 @@ Feature: an user shares resources using ScienceMesh application
                     }
                   }
                 },
+                "eTag": {
+                  "type": "string",
+                  "pattern": "%etag_pattern%"
+                },
```
Comment (ScharfViktor) [inline tests/acceptance/features/apiOcm/share.feature:93]: should we also have “remoteItem” here?
put please "eTag" after "name" to remain consistent

Comment (saw-jan) [reply tests/acceptance/features/apiOcm/share.feature]: >should we also have “remoteItem” here?

IMO, we should

>put please "eTag" after "name" to remain consistent

done

File: tests/acceptance/features/apiOcm/share.feature [hunk 2]
```diff
@@ -45,77 +42,161 @@ Feature: an user shares resources using ScienceMesh application
...
+                  "required": ["user"],
+                  "properties": {
+                    "user": {
+                      "type": "object",
+                      "required": ["displayName", "id", "@libre.graph.userType"],
+                      "properties": {
+                        "displayName": { "const": "Alice Hansen" },
+                        "id": { "pattern": "^%federated_user_id_pattern%$" },
+                        "@libre.graph.userType": { "const": "Federated" }
+                      }
```
Comment (saw-jan) [inline tests/acceptance/features/apiOcm/share.feature:67]: I tried to format for properties checks. still readable and reduce lines

**Pull request merged**